inc/search.class.php

```diff
@@ -1499,7 +1635,7 @@ static function constructData(array &$data, $onlycount = false) {
     *
     * @return void
    **/
-   static function displayData(array $data) {
+   public function displayData(array $data) {
```

Comment:
This method is used in some plugins (`formcreator` and `fusioninventory` for instance), change should be listed in changelog.